### PR TITLE
completed

### PR DESCRIPTION
**Pull request by bug-smith**
completed.
Screenshot 2023-09-15 at 11 39 21 AM

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+console.log(arr);
+console.log('typeof', typeof arr);
+
+const jsonArr = JSON.stringify(arr);
+
+console.log(jsonArr);
+console.log('typeof', typeof jsonArr);
+
+const jsonString = '{"Number id": 456, "name": "Bug Smith"}';
```
Comment (shawnkost) [inline javascript-and-json/main.js]: Properties should always be lowercase, and shouldn't have any spaces. I would change this to just `id`

File: javascript-and-json/main.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (shawnkost) [inline javascript-and-json/main.js:28]: Missing a `JSON.parse(jsonString)` and missing 2 console logs that go with it

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple notes here Bug!

File: javascript-and-json/main.js (hunk 2 above)
Comment (bug-smith) [reply javascript-and-json/main.js:28]: added to a new var, and console logged it, the JSON string is now showing as an object.

Gratzi!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**